src/Sulu/Bundle/CustomUrlBundle/Resources/public/js/components/webspace/settings/custom-url/overlay/main.js

```diff
@@ -206,18 +206,30 @@ define(['underscore', 'text!./form.html'], function(_, form) {
          * Initializes sub-components.
          */
         initializeFormComponents: function() {
-            var routeData = _.filter(
-                _.map(this.data.routes, function(routeDocument, route) {
+            this.customUrls = {};
+
+            var unfilteredRouteData = _.map(this.data.routes, function(routeDocument, route) {
```

Comment:
Why don't you use the [chain method of underscore](http://underscorejs.org/#chain) instead of introducing a variable which is not used afterwards?